src/lib/orionTypes/TypeEntity.cpp

```diff
@@ -116,7 +105,30 @@ std::string TypeEntity::render
     else
     {
       out += valueTag(indent  + "  ", "name", type, ciP->outFormat, true);
-      out += contextAttributeVector.render(ciP, EntityTypes, indent + "  ", false, true);
+      if (ciP->outFormat == JSON) {
+        out += indent + "  \"attributes\": [ ";
+        for (unsigned int ix = 0; ix < contextAttributeVector.size(); ix++)
```

Comment:
This for is duplicated (see lines 84-94). Any refactoring suggestion from the Master of the Renders? :)